src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person}) => {
+  
+  const { name, age, sex, isMarried, partnerName } = person;
+  return (
+  <section className = "Person">
+    <h2 className="Person__name">My name is {name}</h2>
+    {age !== undefined && (
+    <p className="Person__age">I am {age}</p>
+    )}
+
+    {isMarried === true ? (
+       <p className="Person__partner">{partnerName} is my {sex === 'm' ? ('wife') : ('husband')}</p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
```

Comment:
- You don't need to compare with `true` as it's already boolean.
- You have `<p className="Person__partner">` in both cases, only content is different